### DISCUSSION TITLE
blog/2022-09-23-adfree-blog-comments-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#26] [vinnie.work Comments]
# blog/2022-09-23-adfree-blog-comments-giscus/

Background

https://www.vinnie.work/blog/2022-09-23-adfree-blog-comments-giscus/

**Comment by hugo-vrijswijk** [comment]
👋 FYI you can have the comment section theme be reactive to the docusaurus theme, you can just pass the docusaurus theme as a prop to the Giscus react component: https://github.com/stryker-mutator/stryker-mutator.github.io/blob/develop/src/theme/BlogPostItem/Footer/index.tsx#L20-L36

**Reply by crazychenz** [reply]
That's fantastic. Just added the changes. Thank you!

**Comment by RudraSen2** [comment]
Hello, I saw your blog posts at https://www.vinnie.work/blog/2022-09-23-adfree-blog-comments-giscus and the examples that you have taken are my websites, and I recently changed my domain and website name so I request you to update all links in this manner:
- `Comp-Labs` -> `Tech Fiddle`
- `https://complabs.in` -> `https://techfiddle.io`

I'm posting this here because your `edit this page` link is broken. I can't find this post anywhere in this repo. Will appreciate any help.

**Reply by crazychenz** [reply]
sure thing, should be replaced. Do you intend to change your github repo in the near future as well?

**Reply by RudraSen2** [reply]
No, GitHub Repo link would be same. Just the Name and Domain
